apps/web/app/modules/Courses/components/CourseList.tsx

```diff
@@ -3,14 +3,14 @@ import { match } from "ts-pattern";
 import { CardCourseList } from "./CardCourseList";
 import { TableCourseList } from "./TableCourseList";
 
-import type { GetAllCoursesResponse } from "~/api/generated-api";
+import type { GetAvailableCoursesResponse } from "~/api/generated-api";
 import type { CourseListLayout } from "~/types/shared";
 
 export const CourseList: React.FC<{
-  availableCourses: GetAllCoursesResponse["data"];
+  courses: GetAvailableCoursesResponse["data"];
```

Comment:
Can we stay with old name? 
```suggestion
  availableCourses: GetAvailableCoursesResponse["data"];
```